### PR TITLE
Init repo

### PR DESCRIPTION
**Pull request by Aljullu**
This PR creates the plugin scaffold.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+=== WooCommerce Admin ===
```
Comment (ryelle) [inline readme.txt]: Oops, this should probably be WooCommerce Gutenberg Emails :)

File: woocommerce-gutenberg-emails.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * Author: WooCommerce
+ * Author URI: https://woocommerce.com/
+ * Text Domain: woocommerce-gutenberg-emails
+ * Domain Path: /languages
+ * Version: 0.1.0
+ *
+ * WC requires at least: 3.5.0
+ * WC tested up to: 3.5.7
+ *
+ * @package WC_Admin
```
Comment (ryelle) [inline woocommerce-gutenberg-emails.php]: Should this be `WooCommerce Gutenberg Emails`?

File: woocommerce-gutenberg-emails.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+if ( ! defined( 'WC_GUTENBERG_EMAILS_ABSPATH' ) ) {
+	define( 'WC_GUTENBERG_EMAILS_ABSPATH', dirname( __FILE__ ) . '/' );
+}
+
+/**
+ * Calls the loader method
+ *
+ * @see WC_Admin_Library::__construct()
```
Comment (ryelle) [inline woocommerce-gutenberg-emails.php]: Should this be `WC_Gutenberg_Emails_Loader::__construct()`?

**Review by ryelle — APPROVED** [review]
Some WC Admin references still here, but once those are cleaned up this is a great scaffold.

**Comment by Aljullu** [comment]
Thanks for the review @ryelle!

**Pull request merged**